packages/layout/src/util/utils.ts

```diff
@@ -7,12 +10,29 @@ export enum DIRECTION {
     LEFT = "LEFT"
 }
 
+type PositionWithOptionalAnchor = {
+    x: number,
+    y: number,
+    anchor?: true | null,
+}
+
 // https://stackoverflow.com/questions/1026069/how-do-i-make-the-first-letter-of-a-string-uppercase-in-javascript
 export const capitalizeFirstLetter = (string: string): string => {
     return string.charAt(0).toUpperCase() + string.slice(1);
 }
 
 
+export const placeCoordinateOnGrid = (coordinate: number, grid: number): number => {
+    const convertedPosition = coordinate - (coordinate % grid);
+    return convertedPosition;
+}
+
+export const placePositionOnGrid = (position: PositionWithOptionalAnchor, gridX: number, gridY: number): void => {
+    position.x = placeCoordinateOnGrid(position.x, gridX);
+    position.y = placeCoordinateOnGrid(position.y, gridY);
+}
+
```

Comment:
Why are there multiple empty lines?

Reply:
Because the next element is class, so I felt like it makes sense to to have it more separated from the rest.